src/ctype/CType.ts

```diff
@@ -32,7 +34,7 @@ export default class CType implements ICType {
     }
   }
 
-  public async store(identity: Identity) {
+  public async store(identity: Identity): Promise<TxStatus> {
```

Comment:
General remark on the whole PR: this update is super helpful, it's great to be able to see the return types 👍

Reply:
Awesome